packages/flutter_tools/test/general.shard/dart_plugin_test.dart

```diff
@@ -596,18 +592,109 @@ void main() {
               fileSystem: fs,
               appDependencies: directDependencies,
             ),
+            Plugin.fromYaml(
```

Comment:
This test's description says it was supposed to test for multiple errors all being output, but didn't actually do that, so I fixed it.